packages/manager/modules/office/package.json

```diff
@@ -20,8 +20,12 @@
     "@ovh-ux/component-rollup-config": "^6.0.0"
   },
   "peerDependencies": {
-    "@ovh-ux/ovh-utils-angular": "^14.0.2",
+    "@ovh-ux/ng-ovh-swimming-poll": "^4.0.0",
```

Comment:
I cannot find any reference related to this peer dependency inside the Office module.

Could you please confirm me if that dependency is needed?

Thanks!